go-controller/pkg/cni/types.go

```diff
@@ -166,6 +166,9 @@ type PodRequest struct {
 
 	// the DeviceInfo struct
 	deviceInfo nadapi.DeviceInfo
+
+	// indicates if interface is primaryUDN
```

Comment:
Format ?

Reply:
do you mean to lower case IsPrimaryUDN ?